CHANGELOG.md

```diff
@@ -1,40 +1,49 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
-
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+All notable changes to this project will be documented in this file. The project adheres to
+  [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
-### Added
-- Added `patch` method to the `Fork` structure. (#393)
-- Added a public `helthcheck` endpoint. (#405)
-- Added serialization support floating point types through special wrapper(`F32` and `F64`). This feature is hidden behind `float_serialize` gate. Note: special values (Infinity and NaN) aren't supported. (#384)
-
-### Changed
+## 0.5 - 2018-01-29
+
+### Breaking changes
+- The order of bytes and bits in the `DBKey` keys of `ProofMapIndex` became consistent. The change influences how
+  Merkle Patricia trees are built for `ProofMapIndex`: the bits in each byte of a `DBKey` are now enumerated from the
+  least significant bit (LSB) to the most significant bit (MSB), compared to MSB-to-LSB ordering used before. Note:
+  this change will break old storages using map proofs. (#419)
```

Comment:
Should we mention that "to migrate existing storage, create a second exonum instance alongside the old one and move all transactions manually"?

Reply:
I'm not sure: current process is really terrible and I want normal migration as soon as possible. it is controversial, but I don't want such thing in the changelog.